### PR TITLE
Fix some warnings, part 1/x

### PR DESCRIPTION
**Pull request by Kaspik**
This is a long-term project if we want to get into 0 warnings project.

Currently there is more than 300 warnings, here I started with fixing some of them.

### CODE REVIEW

**Comment by Jason-Morcos** [comment]
So we got burned with an earlier fix regarding the moral return codes (FirstButton/Alternate...)
How fully has this been tested?

**Comment by Jason-Morcos** [comment]
See #21 for when we got burned by modals

**Comment by Kaspik** [comment]
If you are talking about `SPUserManager` changes - then I'm not changing the status code, just changing the code style - so I don't think this will cause any issue. :)

**Comment by Jason-Morcos** [comment]
> If you are talking about `SPUserManager` changes - then I'm not changing the status code, just changing the code style - so I don't think this will cause any issue. :)

Even line 803? That was the line that concerned me

File: Source/SPUserManager.m [hunk 1]
```diff
@@ -790,17 +790,18 @@ - (IBAction)doubleClickSchemaPriv:(id)sender
...
+		[alert addButtonWithTitle:NSLocalizedString(@"Continue", @"continue button")];
+		[alert addButtonWithTitle:NSLocalizedString(@"Cancel", @"cancel button")];
 		[alert setAlertStyle:NSWarningAlertStyle];
-		
+
 		// Cancel
-		if ([alert runModal] == NSAlertAlternateReturn) return;
+		if ([alert runModal] == NSAlertFirstButtonReturn) {
+			return;
+		}
```
Comment (Jason-Morcos) [inline Source/SPUserManager.m:804]: This concerns me given we've been burned by changing button codes!

**Comment by Kaspik** [comment]
Yes, the only change there is `return` was moved the new line, no code change itself. 😝

File: Source/SPUserManager.m (hunk 1 above)
Comment (Jason-Morcos) [reply Source/SPUserManager.m:804]: Looks like a change from `NSAlertAlternateReturn` to `NSAlertFirstButtonReturn`?

Comment (Kaspik) [reply Source/SPUserManager.m]: Made a change to simplify stuff + added a docu!

**Comment by Kaspik** [comment]
@Jason-Morcos How we looking here?

**Comment by Jason-Morcos** [comment]
Screen Shot 2020-06-23 at 17 47 05
@Kaspik Still just worried about this change haha. If you tested locally and it works flawlessly, I'm fine with it though

**Comment by Kaspik** [comment]
Yeah I did. Still need your review tho.

**Pull request merged**